components/partials/derivatives/trading/trade.vue

```diff
@@ -324,6 +362,105 @@ export default Vue.extend({
       return this.$accessor.derivatives.marketSummary
     },
 
+    orderTypeToSubmit(): DerivativeOrderSide {
+      const { postOnly, orderTypeBuy } = this
+
+      switch (true) {
+        case postOnly && orderTypeBuy: {
+          return DerivativeOrderSide.BuyPO
+        }
+        case orderTypeBuy: {
+          return DerivativeOrderSide.Buy
+        }
+        case postOnly && !orderTypeBuy: {
+          return DerivativeOrderSide.SellPO
+        }
+        case !orderTypeBuy: {
+          return DerivativeOrderSide.Sell
+        }
+        default: {
+          return DerivativeOrderSide.Buy
+        }
+      }
+    },
+
+    postOnly(): boolean {
```

Comment:
Let's not use computed properties for the purpose of accessing form properties. Same applied for `PortpotionalPercentage` and `reduceOnly`.